uaclient/tests/test_cli_collect_logs.py

```diff
@@ -121,7 +125,7 @@ def test_collect_logs(
             mock.call(["systemctl", "status", "ua.service"], rcs=[0, 3]),
         ]
 
-        assert m_copy.call_count == 14
+        assert m_copy.call_count == 15
```

Comment:
Could we just add here an assertion that `redact_sensitive_logs` is being called as many times as `copy`?